app/src/containers/MenuMobile.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import MenuMobile from '../components/Shared/MenuMobile';
+import { setVisibleMenu } from '../actions/appearence';
+
+const mapStateToProps = (state) => ({
+  menuVisible: state.appearance.menuVisible
```

Comment:
@andresgnlez we can leave this out for now, as clement suggested, but lets create a ticket to improve the overall usage of redux